arch/arm/core/aarch32/cortex_m/fault.c

```diff
@@ -729,6 +730,39 @@ static inline bool z_arm_is_synchronous_svc(z_arch_esf_t *esf)
 	return false;
 }
 
+#if defined(CONFIG_ARMV6_M_ARMV8_M_BASELINE)
+static inline bool z_arm_is_pc_valid(uintptr_t pc)
+{
+	/* Is it in valid text region */
+	extern uint32_t __text_region_start;
+	extern uint32_t __text_region_end;
```

Comment:
why not simply including `include/zephyr/linker/linker-defs.h`?

Reply:
Oh I didn't know about it! Thank you!
I'm traveling this week so it may take a bit of time for the update to come but I'll do that asap!

Reply:
The update should resolve this thread.